README.md

```diff
@@ -65,10 +63,10 @@ Here is an step-by-step example of how to set up a `1deg_jra55do_ryf` experiment
 (called `my_1deg_jra55do_ryf_experiment_name`) after setting up your fork:
```

Comment:
We're not recommending users fork the configs repo, because it is setup for ease of distribution.

Our recommended workflow is covered in the Hive Docs (in this case for ACCESS-OM2):

https://access-hive.org.au/models/run-a-model/run-access-om/#get-access-om2-configuration

In the absence of a Hive Docs run the model page, I'd suggest something like:
```
payu clone -b expt_name -B release-1deg_jra55do_ryf https://github.com/ACCESS-NRI/access-om2-configs 1deg_jra55do_ryf
```

which clones into a directory called `deg_jra55do_ryf` and checks out an experiment branch called `expt_name`.

We also need to remove the reference to COSIMA in this context.

Reply:
This is a good suggestion (TM) but unrelated to this PR :)

Reply:
Yeah I think this suggestion should be moved into a separate issue about the README.md - I only added the bottom bit.